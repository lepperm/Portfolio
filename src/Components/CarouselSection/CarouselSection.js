/**
 * @fileoverview CarouselSection component.
 */

import React from "react";
import PropTypes from "prop-types";

import { ProjectCard } from "../ProjectCard";

/**
 * CarouselSection component
 *
 * @return {Component} CarouselSection component
 */
export const CarouselSection = () => {
  return (
    <section id="carousel-section">
      <p>Project cards scrolling here</p>
      <ul>
        <li key="indexvalue">
          <p>
            Given an object of X elements, map each element into a template list
            item
          </p>
          <ProjectCard />
        </li>
      </ul>
      <p>
        Maybe a progress bar to show how many projects are in the list and where
        in the list is focused, clickable to jump to entry?
      </p>
    </section>
  );
};

/**
 * Prop types for CarouselSection component
 */
CarouselSection.propTypes = {
  //
};
