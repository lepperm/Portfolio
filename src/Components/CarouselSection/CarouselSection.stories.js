/**
 * @fileoverview Story for CarouselSection component.
 */

import React from "react";
import { CarouselSection } from "./CarouselSection";

export default {
  title: "CarouselSection",
  component: CarouselSection,
};

export const Basic = () => <CarouselSection />;
