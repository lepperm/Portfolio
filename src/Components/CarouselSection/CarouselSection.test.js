/**
 * @fileoverview Tests for CarouselSection component.
 */

import React from "react";
import { shallow } from "enzyme";
import { CarouselSection } from "./CarouselSection";

describe("CarouselSection", () => {
  test("matches snapshot", () => {
    const wrapper = shallow(<CarouselSection />);
    expect(wrapper).toMatchSnapshot();
  });
});
