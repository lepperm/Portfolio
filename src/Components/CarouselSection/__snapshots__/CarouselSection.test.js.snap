// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CarouselSection matches snapshot 1`] = `ShallowWrapper {}`;
